{
  "channeldata": {
    "activate.d": false,
    "binary_prefix": false,
    "deactivate.d": false,
    "home": "https://github.com/testing-cabal/traceback2",
    "license": "PSF 2.0",
    "post_link": false,
    "pre_link": false,
    "pre_unlink": false,
    "run_exports": {},
    "source_url": "https://pypi.io/packages/source/t/traceback2/traceback2-1.4.0.tar.gz",
    "subdirs": [
      "linux-64"
    ],
    "summary": "Backports of the traceback module",
    "text_prefix": false,
    "version": "1.4.0"
  },
  "channeldata_version": 1,
  "feedstock": null,
  "labels": [
    "main"
  ],
  "package": "traceback2-1.4.0-py35_0.tar.bz2",
  "repodata": {
    "arch": "x86_64",
    "build": "py35_0",
    "build_number": 0,
    "constrains": [
      "python_abi * *_cp35m"
    ],
    "depends": [
      "linecache2",
      "python 3.5*",
      "six"
    ],
    "license": "PSF 2.0",
    "md5": "2ebd78b991e831155ff1d42f5aea8bcc",
    "name": "traceback2",
    "platform": "linux",
    "sha256": "13185ae0fac672e445be6e3fb4f1d70ed1536152fffcaf82224db6a19800241f",
    "size": 243365,
    "subdir": "linux-64",
    "version": "1.4.0"
  },
  "repodata_version": 1,
  "subdir": "linux-64",
  "url": "https://conda.anaconda.org/conda-forge/linux-64/traceback2-1.4.0-py35_0.tar.bz2"
}